Neos.Media/Configuration/Settings.yaml

```diff
@@ -56,6 +56,10 @@ Neos:
 
     thumbnailGenerators:
 
+      Neos\Media\Domain\Model\ThumbnailGenerator\ConvertImageThumbnailGenerator:
```

Comment:
Interesting, I didn't know that this exists. But as far as I understand, it is only possible to use this class once. So it is only possible to convert a set of extensions to one other - it wouldn't be possible to add another rule for gif -> png?

Reply:
Yes you can only have one default image format non web images are converted to. I still prefer this solution to adding new concepts orconfigurations.

Reply:
@daniellienert btw, this did not exist before i extended the ImageThumbnailGenerator in a way that it will always specify a target format. That is why it had to be in here since format was previously not cared about.